supporter-product-data/cloudformation/cfn.yaml

```diff
@@ -319,6 +352,63 @@ Resources:
                   - states:StartExecution
                 Resource: !Ref SupporterProductData
 
+  SupporterProductDataDeadLetterQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub dead-letters-supporter-product-data-${Stage}
+      Tags:
+        - Key: Stage
+          Value: !Ref Stage
+
+  SupporterProductDataQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub supporter-product-data-${Stage}
+      RedrivePolicy:
+        deadLetterTargetArn:
+          Fn::GetAtt:
+            - "SupporterProductDataDeadLetterQueue"
+            - "Arn"
+        maxReceiveCount: 10
+      VisibilityTimeout: 3000
+      Tags:
+        - Key: Stage
+          Value: !Ref Stage
+
+  SQSTrigger:
+    Type: AWS::Lambda::EventSourceMapping
+    Properties:
+      BatchSize: 1
```

Comment:
any particular reason for a batch size so small or just for simplicity?  I'm sure it's not a big deal, but I'm wondering the reason.

Reply:
Just copied and pasted from elsewhere and didn't change it. I'm not sure if there are any real advantages to a larger batch size but I've upped it to 10